js/src/get-started-page/get-started-card/index.scss

```diff
@@ -1,41 +1,60 @@
-.woocommerce-marketing-google-get-started-card {
-	.components-flex {
-		flex-direction: column;
-		align-items: flex-end;
+@import "../_shared";
 
-		@include break-large() {
-			flex-direction: row;
-			align-items: center;
-		}
+.gla-get-started-card {
+	display: flex;
+	flex-direction: row-reverse;
+	padding: $grid-unit-50 $grid-unit-50 $grid-unit-20 $grid-unit-50;
```

Comment:
💅  The last `$grid-unit-50` could be omitted when it's the same as the second one.

Reply:
Thanks! Updated in a819c2e.